server/api/tumdev/campus_backend.proto

```diff
@@ -376,6 +383,27 @@ message Room {
   string name = 9;
 }
 
+message NewsItem{
+  int32 id = 1;
+  string title = 2;
+  string text = 3;
+  string link = 4;
+  string image_url = 5;
+  string source = 6;
+  google.protobuf.Timestamp created = 7;
+}
+
+message GetNewsReply {
+  repeated NewsItem news = 1;
+}
+
+message GetNewsRequest {
+  // the last id of the news item received. 0 to get all news items
+  int32 last_news_id = 1;
+  // filter by news source id. 0 or empty to get all news items
```

Comment:
nit: "empty" is not an option. Unset might be.

Reply:
Given that it is unclear how other programing languages implment unset values, lets just document 0

```suggestion
  // filter by news source id. 0 to get all news items
```